[debug] [2020-08-09T14:26:33.737Z] ----------------------------------------------------------------------
[debug] [2020-08-09T14:26:33.739Z] Command:       /usr/local/bin/firebase /Users/saumyasinghal/.cache/firebase/tools/lib/node_modules/firebase-tools/lib/bin/firebase serve
[debug] [2020-08-09T14:26:33.739Z] CLI Version:   8.7.0
[debug] [2020-08-09T14:26:33.739Z] Platform:      darwin
[debug] [2020-08-09T14:26:33.739Z] Node Version:  v12.18.1
[debug] [2020-08-09T14:26:33.740Z] Time:          Sun Aug 09 2020 07:26:33 GMT-0700 (Pacific Daylight Time)
[debug] [2020-08-09T14:26:33.740Z] ----------------------------------------------------------------------
[debug] [2020-08-09T14:26:33.740Z] 
[debug] [2020-08-09T14:26:33.754Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-08-09T14:26:33.754Z] > authorizing via signed-in user
[debug] [2020-08-09T14:26:33.754Z] [iam] checking project socrathematics for permissions ["firebase.projects.get"]
[debug] [2020-08-09T14:26:33.757Z] > refreshing access token with scopes: ["email","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","openid"]
[debug] [2020-08-09T14:26:33.757Z] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token  
 <request body omitted>
[debug] [2020-08-09T14:26:33.900Z] <<< HTTP RESPONSE 200 {"content-type":"application/json; charset=utf-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","date":"Sun, 09 Aug 2020 14:26:33 GMT","server":"scaffolding on HTTPServer2","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3-29=\":443\"; ma=2592000,h3-27=\":443\"; ma=2592000,h3-T050=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2020-08-09T14:26:33.923Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/socrathematics:testIamPermissions  
 {"permissions":["firebase.projects.get"]}
[debug] [2020-08-09T14:26:34.145Z] <<< HTTP RESPONSE 200 {"content-type":"application/json; charset=UTF-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","date":"Sun, 09 Aug 2020 14:26:34 GMT","server":"ESF","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","server-timing":"gfet4t7; dur=142","alt-svc":"h3-29=\":443\"; ma=2592000,h3-27=\":443\"; ma=2592000,h3-T050=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2020-08-09T14:26:34.147Z] >>> HTTP REQUEST GET https://firebase.googleapis.com/v1beta1/projects/socrathematics  
 
[debug] [2020-08-09T14:26:34.356Z] <<< HTTP RESPONSE 200 {"content-type":"application/json; charset=UTF-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","date":"Sun, 09 Aug 2020 14:26:34 GMT","server":"ESF","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3-29=\":443\"; ma=2592000,h3-27=\":443\"; ma=2592000,h3-T050=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[info] 
[info] === Serving from '/Users/saumyasinghal/Documents/GitHub/ymath.io'...
[info] 
[debug] [2020-08-09T14:26:34.361Z] TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at validateString (internal/validators.js:120:11)
    at Object.join (path.js:1039:7)
    at Object.<anonymous> (/Users/saumyasinghal/.cache/firebase/tools/lib/node_modules/firebase-tools/lib/serve/functions.js:22:39)
    at Generator.next (<anonymous>)
    at /Users/saumyasinghal/.cache/firebase/tools/lib/node_modules/firebase-tools/lib/serve/functions.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/saumyasinghal/.cache/firebase/tools/lib/node_modules/firebase-tools/lib/serve/functions.js:4:12)
    at Object.start (/Users/saumyasinghal/.cache/firebase/tools/lib/node_modules/firebase-tools/lib/serve/functions.js:20:16)
    at /Users/saumyasinghal/.cache/firebase/tools/lib/node_modules/firebase-tools/lib/serve/index.js:24:40
    at arrayMap (/Users/saumyasinghal/.cache/firebase/tools/lib/node_modules/lodash/lodash.js:639:23)
    at Function.map (/Users/saumyasinghal/.cache/firebase/tools/lib/node_modules/lodash/lodash.js:9580:14)
    at /Users/saumyasinghal/.cache/firebase/tools/lib/node_modules/firebase-tools/lib/serve/index.js:23:29
    at Generator.next (<anonymous>)
    at /Users/saumyasinghal/.cache/firebase/tools/lib/node_modules/firebase-tools/lib/serve/index.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/saumyasinghal/.cache/firebase/tools/lib/node_modules/firebase-tools/lib/serve/index.js:4:12)
[error] 
[error] Error: An unexpected error has occurred.
